### PR TITLE
Order Users By No. of Posts/Comment

### PR DESCRIPTION
**Pull request by AhmedKamal20**

### CODE REVIEW

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -1,3 +1,10 @@
 class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
+  before_action :set_page_number
+
+  private
+
+  def set_page_number
+    params[:page] = params[:page].present? ? params[:page].to_i : 1
+  end
```
Comment (emad-elsaid) [inline app/controllers/application_controller.rb]: Nop, we shouldn't have any logic related to pagination in the controller, anything related to pagination should be passed to the service from the controller, that logic can lead to complexities in the future.

**Comment by emad-elsaid** [comment]
That looks good, just one thing to keep the `ApplicationController` clean

**Comment by AhmedKamal20** [comment]
@emad-elsaid Check last commit, Please

**Comment by emad-elsaid** [comment]
Any last thoughts before merging?

**Comment by emad-elsaid** [comment]
I also created 2 Labels for "work in progress" and "ready for review"

**Comment by AhmedKamal20** [comment]
no, go ahead and merge if you are OK with it 
BTW, i can't add labels to issues here

**Comment by emad-elsaid** [comment]
nah, I created it becaue "WIP" in the title, i thought changing the title over time isn't a good thing to do and labels are the way to go in that case.
thanks for that feature man 👍 it feels good when we're collaborating on this project

**Pull request merged**